app/helpers/application_helper.rb

```diff
@@ -18,9 +18,10 @@ def document_leaflet_map(document:, **)
 
   ##
   # @param [String] manifest
-  def iiif_drag_n_drop(manifest, width: '40')
+  def iiif_drag_n_drop(manifest, width: '40', document: nil)
     link_url = format Settings.iiif_dnd_base_url, query: { manifest: manifest }.to_query
-    link_to link_url, class: 'iiif-dnd float-right', data: { turbolinks: false } do
+    link_to link_url, class: 'iiif-dnd float-right', data: { turbolinks: false },
+                      aria: { label: document['title_display'] } do
```

Comment:
Shouldn't this be something like "Drag to a IIIF viewer to open \"#{document['title_display']}\"" ?

Reply:
The ticket indicates no? ```<div class="col-2 col-md-1">
  <a class="iiif-dnd float-right" data-turbolinks="false" href=..." aria-label=" Cambridge, Corpus Christi College, MS 002I: The Bury Bible">
    <img alt="IIIF Drag-n-drop" src="..." width="30">
  </a>
</div>
```

Reply:
@jcoyne I talked with Alan. We have updated the aria label to include drag-n-drop and made the alt text empty.